pom.xml

```diff
@@ -549,7 +551,52 @@
         <version>5.17.2</version>
       </dependency>
 
-      <!-- Httpcomponents -->
+      <dependency>
```

Comment:
Why do we need these deps? Conceptually our integ-test Is already working with existing bundles and existing dependencies.. we should not need any changes to add the test-suite

Reply:
These caused some dependency issues when running tests. I will look into this shortly.

Reply:
Addressed, removed the changes.